git/gitgithubclient.py

```diff
@@ -40,7 +39,7 @@ def _commitData(self, commit):
 			"message": commit.commit.message
 		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since=0, to=0):
```

Comment:
The ``commits`` function signature says ``to=0``. Inside the function it is changed to the current date + 24 hours. This will be confusing for users. Any ideas how to make the date format transparent? I can think of introducing new function ``commitsSince`` which will take only ``since`` parameter and ``commitsTo`` which will take only ``to`` parameter. So when calling ``commits`` caller will always have to specify both ``since`` and ``to`` parametr. Both ``commitsSince`` and ``commitsFrom`` would call ``commits`` and carry required conversions.